In order to test the calib code in simulation with the proper environment, we do the following from mfx-monitor:

First, since we work in a local clone:
```bash
cd mfx
source mfxenv
hutch-python --cfg conf.yml
```

```python
In [1]: from mfx.optimize.beamline_hw import sim_devices

In [2]: sim_devices()

In [3]: from mfx.optimize.beam import Beam

In [4]: beam = Beam()

In [5]: beam.align(using_device="yag", on_diagnostic="xcs1", mover="und", use_2d_markers=True, with_method="calib")
```

The current behavior is: 

```python

In [3]: from mfx.optimize.beam import Beam
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 from mfx.optimize.beam import Beam

File ~/mfx/mfx/optimize/beam.py:18, in <module>
     16 from .user_select import select_diagnostic, select_goal, MP_KEY, UNDP_KEY_X, UNDP_KEY_Y
     17 from .constraints import constraint_data
---> 18 from .utils import snake_order
     20 class Beam:
     22     def _predict_centroid(self, undp_xy: tuple[float, float], calib: dict) -> tuple[float, float]:

File ~/mfx/mfx/optimize/utils.py:10, in <module>
      8 import torch
      9 import pandas as pd
---> 10 from lcls_tools.common.image.fit import ImageProjectionFit
     12 from .constraints import constraint_data
     13 from .beamline_hw import sim_devices

ModuleNotFoundError: No module named 'lcls_tools'
```

--- a/mfx/optimize/xopt_scans.py
+++ b/mfx/optimize/xopt_scans.py
@@ -383,7 +383,7 @@ def get_xopt_obj(
     #vocs.constraints = {}
     if device_type == "yag":
         # Create per run images directory
-        images_root = Path("/cds/home/opr/mfxopr")
+        images_root = Path.home() #Path("/cds/home/opr/mfxopr")
         images_root.mkdir(parents=True, exist_ok=True)
         run_dir_name = Path(dump_file).stem
         run_images_dir = images_root / run_dir_name
@@ -451,10 +451,10 @@ def get_xopt_obj(
 
 def test_write_permissions():
     """
-    Simple test function to check if we can write to /cds/home/opr/mfxopr
+    Simple test function to check if we can write to /cds/home/opr/mfxopr (or your home)
     """
 
-    test_dir = Path("/cds/home/opr/mfxopr")
+    test_dir = Path.home() #Path("/cds/home/opr/mfxopr")
     test_dir.mkdir(parents=True, exist_ok=True)
     test_file = test_dir / f"test_write.txt"
 

